doc/test-framework/writing-e2e-tests.md

```diff
@@ -10,19 +10,19 @@ as standard go tests.
 ## Components
 
 The test framework includes a few components. The most important to talk
-about are Framework and TestCtx.
+about are Framework and Context.
 
 ### Framework
 
 [Framework][framework-link] contains all global variables, such as the kubeconfig, kubeclient,
 scheme, and dynamic client (provided via the controller-runtime project).
 It is initialized by MainEntry and can be used anywhere in the tests.
 
-### TestCtx
+### Context
```

Comment:
It is an break change. The options here would call it as Ctx or Context. So, I am looking for the team position as well.

Reply:
I think my position is that we should make a first pass through the linter suggestions to make all the non-breaking change fixes that we can.

IMO, we should not introduce breaking changes for the sake of passing a linter check only.

Reply:
But thes test.TextCtx is not accepted by the linter since is not a good practice to use the name of the pkg in the struct because it will be called a test.TextCtx. So what is your suggestion?

Reply:
Another way could be to keep test.TextCtx and add // #nolint for it be ignored. However, in this case we can do the fix so I think we should move forward and change the name in order to be in accordance with.